demo/site/src/app/block-preview/page/page.tsx

```diff
@@ -1,11 +1,35 @@
 "use client";
-import { BlockPreviewProvider, IFrameBridgeProvider, useIFrameBridge } from "@comet/cms-site";
+import { BlockPreviewProvider, IFrameBridgeProvider, useBlockPreviewFetch, useIFrameBridge } from "@comet/cms-site";
+import { PageContentBlockData } from "@src/blocks.generated";
 import { PageContentBlock } from "@src/blocks/PageContentBlock";
+import { recursivelyLoadBlockData } from "@src/recursivelyLoadBlockData";
+import { graphQLApiUrl } from "@src/util/graphQLClient";
 import * as React from "react";
 
 const PreviewPage: React.FunctionComponent = () => {
     const iFrameBridge = useIFrameBridge();
-    return <div>{iFrameBridge.block && <PageContentBlock data={iFrameBridge.block} />}</div>;
+
+    const { fetch, graphQLFetch } = useBlockPreviewFetch(graphQLApiUrl);
```

Comment:
@raphaelblum what do you think about the distinction between fetch (REST) and graphQLFetch (GraphQL)? Is this clear enough?

Reply:
for me it is, yes.